analyzer/runtime/runtime.go

```diff
@@ -318,6 +319,23 @@ func (m *Main) queueDbUpdates(batch *storage.QueryBatch, data *BlockData) {
 			error_code,
 			error_message,
 		)
+
+		// Handle native-token transfers from transactions.
+		// These transfers do not emit events and are not covered
+		// by the accounts module, so we process them here.
+		if transactionData.Method == "evm.Call" {
+			body := transactionData.Body.(*evm.Call)
+			if len(body.Data) == 0 {
+				amount := common.QuantityFromBytes(body.Value)
+				batch.Queue(
```

Comment:
can we do that here https://github.com/oasisprotocol/oasis-indexer/blob/316cd3b3bf5eaa217e026d33c9a5086bb98307d5/analyzer/runtime/extract.go#L382

Reply:
+1, the runtime analyzer is more structured than the consensus analyzer, and tends to first store all the analysis results into a big struct, then convert that struct into sql statements in a separate step.

Reply:
updated!